toolkit/scripts/pkggen.mk

```diff
@@ -104,10 +99,10 @@ else
 graphpkgfetcher_update_repo_flag :=
 endif
 # Compare files via checksum (-c) instead of timestamp so unchanged RPMs are left intact without updating the timestamp of the directories
```

Comment:
Can you nuke this comment while you are at it? We aren't using rsync anywhere now